scripts/set_xva_bridge.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# functions
+# usage
+usage()
+{
+    echo "--------------------------------------------------------------------------------"
+    echo "- Usage: $0 [XVA's_filename] compression[zstd|gzip] <bridge_value>"
+    echo "- First and second options are mandatory."
+    echo "- The third option is optional. if nothing is filled, the value will be xenbr0."
+    echo "--------------------------------------------------------------------------------"
+}
+
+# extract and read ova.xml
+info_ovaxml()
+{
+    TMPFOLD=$(mktemp -d /tmp/xvaXXXX)
+    tar -xvf $nameXVA -C $TMPFOLD ova.xml
+    chmod +r $TMPFOLD/ova.xml
+    MyXmlValue=$(grep -oE "<member><name>bridge</name><value>[^<]*</value></member>" $TMPFOLD/ova.xml)
+    MyLenght=${#MyXmlValue}
+    MyLenghtMinus=$(($MyLenght-17))
+    MyNetworkValue=$(cut -c 1-$MyLenghtMinus <<< $MyXmlValue)
+    BridgeNetwork=$(cut -c 35-${#MyNetworkValue} <<< $MyNetworkValue)
+
+    if [ -d $TMPFOLD ]; then
+        rm -Rf $TMPFOLD
+    fi
+}
+
+# check parameters and prompt the usage if needed
+if [ -z "$1" ]
+then
+    echo "xva's name missing!"
+    usage
+    exit 1
+else
+    if [ -e $1 ]; then
+        nameXVA=$(realpath $1)
+
+        if [ ${nameXVA##*.} !=  xva ]
+        then
+            echo "Error: $nameXVA doesn't seem to be a xva file"
+            usage
+            exit 1
+        fi
+    else
+        echo "Error: $1 doesn't seem to exist as a file. Is the path or the name are corrects?"
+        usage
+        exit 1
+    fi
+fi
+
+if ([ "$2" = "gzip" ] || [ "$2" = "zstd" ])
+then
+    CompressM=$2
+else
+    echo "Error: Wrong option for the compression."
+    usage
+    exit 1
+fi
+
+if [ -z "$3" ]
+then
+    BridgeValue="xenbr0"
+else
+    BridgeValue=$3
+fi
+
+# we want to know the value of the network bridge.
+# then we can decide if we need to change it or not.
+info_ovaxml
+
+if [ -z $BridgeNetwork ]
```

Comment:
It's not really clean nor readable to rely on a global variable defined by the function you called.

It would be better if `invo_ovaxml` would echo the result, and to write:
```
BRIDGE=$(info_ovalxml)
```

By the way maybe `info_ovaxml` should just be called `get_bridge`.